small-talk/app/chat/page.js

```diff
@@ -5,159 +5,128 @@ import socket from "../../util/socket";
-    };
-
-	return (
-        <ThemeLayout>
-            <div // all the styles added in this file are temporary for testing purposes
-				data-testid="chat-container" 
-				style={{ 
-					display: 'flex',
-					flexDirection: 'column',
-					alignItems: 'center', // Align items to the center horizontally
-					backgroundColor: 'pink',
-					color: 'black',
-					padding: '20px',
-					borderRadius: '10px',
-					maxWidth: '80%', // Limit the maximum width of the container
-					margin: '20vh auto 0', // Move the container down by 20% of viewport height
-					height: '70vh', // Set the height to 70% of the viewport height
-					maxHeight: '600px', // Limit the maximum height to 600px
-					overflow: 'hidden', // Hide overflow to prevent scrollbars on the container itself
-					boxShadow: '0 0 10px rgba(0, 0, 0, 0.1)', // Add a subtle shadow for depth
-            	}}>
-
-				<h2>Private Chat</h2>
-				<div 
-					className="private-messages-list"
-					style={{
-						height: 'calc(100% - 220px)', // Set the height of the message container dynamically
-						width: '100%',
-						overflowY: 'auto', // Allow vertical scrolling if content overflows
-					}}>
-
-					{privateMessages.map((message, index) => (
-						<div key={index}>
-						<strong>{message.from === socket.id ? "You" : "Recipient"}:</strong> {message.content}
-						</div>
-					))}
-				</div>
-
-				<h3 style = {{ color:'blue' }}>Users:</h3>
-				<ul className="user-list" style={{ color: 'purple', padding: 0, margin: 0, maxHeight: '120px', overflowY: 'auto' }}>
-					{user.map(user => (
-						<li key={user.userID} onClick={() => setSelectedUser(user)}>
-							{user.username}
-						</li>
-					))}
-				</ul>
+		sendMessage(inputMessage, selectedUser, setPrivateMessages, setInputMessage);
+	};
 
-				<input
-					type="text"
-					value={inputMessage}
-					onChange={(e) => setInputMessage(e.target.value)}
-					placeholder="Type your message here..."
-					style={{ width: '100%', marginBottom: '10px' }} // Set the width to 100% and add some bottom margin
-				/>
-				<div style={{ display: 'flex', justifyContent: 'center' }}>
-					<button style={{ backgroundColor: 'yellow', marginRight: '10px' }} onClick={handleConnectButtonClick}>Connect</button>
-					<button style={{ backgroundColor: 'green', marginRight: '10px' }} onClick={handleSendMessage}>Send</button>
-				</div>
-
-            </div>
+	const onUsernameSelection = (username) => {
+		socket.auth = { username };
+		socket.connect();
+	};
 
-        </ThemeLayout>
+	return (
+		<ThemeLayout>
+		<div
+			data-testid="chat-container" 
+			style={{ 
+				display: 'flex',
+				flexDirection: 'column',
+				alignItems: 'center',
+				backgroundColor: 'pink',
+				color: 'black',
+				padding: '20px',
+				borderRadius: '10px',
+				maxWidth: '80%',
+				margin: '20vh auto 0',
+				height: '70vh',
+				maxHeight: '600px',
+				overflow: 'hidden',
+				boxShadow: '0 0 10px rgba(0, 0, 0, 0.1)',
+			}}>
```

Comment:
For consistency we should be using Tailwind CSS as much as possible

Reply:
All the styling needs to be fixed later on for consistency, but it would probably be the last thing we are going to focus on and after all the other chatroom issues.